### PR TITLE
TL translation - 1,108 words

### PR DESCRIPTION
**Pull request by dranem**
Update guis.asc
Update eclipse.asc
Update command-line.asc
Update 1-embedding-git.asc 
Update powershell.asc

### CODE REVIEW

File: book/A-git-in-other-environments/sections/guis.asc [hunk 1]
```diff
@@ -28,7 +28,7 @@ $ gitk [git log options]
 ----
 
 Tinatanggap ng Gitk ang maraming mga opsyon sa command line, na karamihan ay naipasa sa pinagbabatayan na aksyon ng `git log`.
-Marahil ang isa sa mga pinaka-kapaki-pakinabang ay ang `--all` na flag, na nagsasabi sa gitk na ipakita ang mga commit na mula sa _any_ ref, hindi lamang HEAD.
+Marahil ang isa sa mga pinaka-kapaki-pakinabang ay ang `--all` na flag, na nagsasabi sa gitk na ipakita ang mga commit na mula sa _anumang_ ref, hindi lamang HEAD.
```
Comment (mind-blast) [inline book/A-git-in-other-environments/sections/guis.asc]: gitk to git 
hindi lamang HEAD to na hindi lamang HEAD

File: book/A-git-in-other-environments/sections/eclipse.asc [hunk 2]
```diff
@@ -1,10 +1,10 @@
-=== Git in Eclipse
+=== Git sa Eclipse
 
 (((Eclipse)))
-Eclipse ships with a plugin called Egit, which provides a fairly-complete interface to Git operations.
-It's accessed by switching to the Git Perspective (Window > Open Perspective > Other…, and select "Git").
+Ang eclipse ay may kasamang isang plugin na tinatawag na Egit, na nagbibigay ng pantay-kumpletong interface sa mga operasyon ng Git.
+Na-access ito sa pamamagitan ng paglipat sa Git Perspective (Window> Buksan ang Perspective> Iba pa..., at piliin ang "Git").
```
Comment (mind-blast) [inline book/A-git-in-other-environments/sections/eclipse.asc]: Perspective to Pananaw

File: book/A-git-in-other-environments/sections/guis.asc [hunk 3]
```diff
@@ -8,14 +8,14 @@ Ngunit ang plain text ay hindi ang pinakamahusay na pagpipilian para sa lahat ng
 Mahalagang tandaan na ang iba't ibang mga interface ay natatangi para sa iba't-ibang mga workflow.
 Ang ilang mga kliyente ay naglalantad lamang ng maingat na na-curate na subset ng Git na functionality, upang suportahan ang isang partikular na paraan ng paggawa na itinuturing ng may-akda na epektibo.
 Kapag tiningnan sa liwanag na ito, wala sa mga kasangkapan na ito ay maaaring masabi na ``mas mabuti'' kaysa sa iba pa, ang mga ito ay mas angkop sa kanilang layunin.
-Tandaan din na wala sa mga graphical na kliyente na maaaring gawin na ang command-line kliyente ay hindi kaya; ang command-line ay pa rin kung saan magkakaroon ka ng pinaka makapangyarihan at kontrol kapag gumagawa sa iyong mga repository.
+Tandaan din na wala sa mga grapikal na kliyente na maaaring gawin na ang command-line kliyente ay hindi kaya; ang command-line ay pa rin kung saan magkakaroon ka ng pinaka makapangyarihan at kontrol kapag gumagawa sa iyong mga repository.
```
Comment (mind-blast) [inline book/A-git-in-other-environments/sections/guis.asc]: command-line kliyente to command-line na kliyente
check this sentence :  ang command-line ay pa rin kung saan

File: book/A-git-in-other-environments/sections/guis.asc [hunk 4]
```diff
@@ -105,46 +105,37 @@ Tinutulungan nito na isama ang isang bundle na bersyon ng Git, na nangangahuluga
...
 
-===== Recommended Workflow
+===== Inirerekumendang Workflow
 
-Once it's installed and configured, you can use the GitHub client for many common Git tasks.
-The intended workflow for this tool is sometimes called the ``GitHub Flow.''
-We cover this in more detail in <<_github#_github_flow>>, but the general gist is that (a) you'll be committing to a branch, and (b) you'll be syncing up with a remote repository fairly regularly.
+Sa sandaling naka-install at nakompigura, maaari mong gamitin ang GitHub na kliyente para sa maraming karaniwang mga commit sa Git. Ang inilaan na workflow para sa kasangkapan na ito kung minsan ay tinatawag na ``GitHub Flow.'' Sinasaklaw namin ito nang mas detalyado sa <<_github#_github_flow>>, ngunit ang pangkalahatang gist ay (a) magkakapag-commit ka sa isang branch, at (b) makakapag-sync ka ng isang remote na repositoryo ng madalas.
```
Comment (mind-blast) [inline book/A-git-in-other-environments/sections/guis.asc]: maaari to ay maaari

File: book/A-git-in-other-environments/sections/guis.asc [hunk 5]
```diff
@@ -105,46 +105,37 @@ Tinutulungan nito na isama ang isang bundle na bersyon ng Git, na nangangahuluga
...
 
-.Creating a branch on Windows.
-image::images/branch_widget_win.png[Creating a branch on Windows.]
+.Paglikha ng isang branch sa Windows
+image::images/branch_widget_win.png[Paglikha ng isang branch sa Windows.]
 
-Once your branch is created, making new commits is fairly straightforward.
-Make some changes in your working directory, and when you switch to the GitHub client window, it will show you which files changed.
-Enter a commit message, select the files you'd like to include, and click the ``Commit'' button (ctrl-enter or ⌘-enter).
+Sa sandaling ang iyong branch ay nilikha, ang paggawa ng mga bagong commit ay medyo dalisay.
```
Comment (mind-blast) [inline book/A-git-in-other-environments/sections/guis.asc]: dalisay to tahas

File: book/A-git-in-other-environments/sections/guis.asc [hunk 6]
```diff
@@ -105,46 +105,37 @@ Tinutulungan nito na isama ang isang bundle na bersyon ng Git, na nangangahuluga
...
+Gumawa ng ilang mga pagbabago sa iyong gumaganang direktoryo, at kapag lumipat ka sa window ng kliyente ng GitHub, ipapakita nito sa iyo kung aling mga file ang nagbago. Maglagay ng isang commit ng mensahe, piliin ang mga file na nais mong isama, at pindotin ang ``Commit'' na pindutan (ctrl-enter o ⌘-enter).
```
Comment (mind-blast) [inline book/A-git-in-other-environments/sections/guis.asc]: pindotin  to pindutin

File: book/A-git-in-other-environments/sections/powershell.asc [hunk 7]
```diff
@@ -1,18 +1,16 @@
...
 
 .Powershell with Posh-git.
 image::images/posh-git.png[Powershell with Posh-git.]
 
-==== Installation
-===== Prerequisites
-Before you're able to run PowerShell scripts on your machine, you need to set your local ExecutionPolicy to RemoteSigned (Basically anything except Undefined and Restricted). If you choose AllSigned instead of RemoteSigned, also local scripts (your own) need to be digitally signed in order to be executed. With RemoteSigned, only Scripts having the "ZoneIdentifier" set to Internet (were downloaded from the web) need to be signed, others not.
+==== Pag-install
+===== Mga kinakailangan
+Bago mo magagawang patakbuhin ang mga script ng PowerShell sa iyong makina, kailangan mong itakda ang iyong lokal na ExecutionPolicy sa RemoteSigned (Karaniwang anumang bagay maliban sa Hindi tiyak and Ipinagbabawal). If you choose AllSigned instead of RemoteSigned, also local scripts (your own) need to be digitally signed in order to be executed. With RemoteSigned, only Scripts having the "ZoneIdentifier" set to Internet (were downloaded from the web) need to be signed, others not.
```
Comment (mind-blast) [inline book/A-git-in-other-environments/sections/powershell.asc]: and  to at

**Comment by dranem** [comment]
Update all the changes commented by robin-ho

**Pull request merged**